dotnet.cmd

```diff
@@ -16,7 +16,7 @@ set Platform=
 :: Don't resolve runtime, shared framework, or SDK from other locations to ensure build determinism
 set DOTNET_MULTILEVEL_LOOKUP=0
 
-:: Disable first run since we want to control all package sources
-set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
```

Comment:
This was replaced by `DOTNET_NOLOGO` in .NET Core 3 which is set in tools.sh/ps1.